source/install.sh

```diff
@@ -2,12 +2,36 @@
 SCRIPT=$(readlink -f "${BASH_SOURCE[0]}")
 SOURCE_PATH=$(dirname "$SCRIPT")
 
-# options
-# TODO: parse arguments and provide --help to set these options
-BUILD_TESTING="OFF"
 BUILD_CONTROLLERS="ON"
 BUILD_DYNAMICAL_SYSTEMS="ON"
 BUILD_ROBOT_MODEL="ON"
+BUILD_TESTING="OFF"
+
+FAIL_MESSAGE="The provided input arguments are not valid.
+Run the script with the '--help' argument."
+
+HELP_MESSAGE="Usage: ./install.sh [OPTIONS]
+
+An install script for the control libraries.
+
+Options:
+  --no-controllers         Exclude the controllers library
+  --no-dynamical-systems   Exclude the dynamical systems library
+  --no-robot-model         Exclude the robot model library
+  --build-tests            Build the unittest targets
+  --help                   Show this help message"
+
+while [ "$#" -gt 0 ]; do
+  case "$1" in
+    --no-controllers=*) BUILD_CONTROLLERS="OFF"; shift 1;;
+    --no-dynamical-systems=*) BUILD_DYNAMICAL_SYSTEMS="OFF"; shift 1;;
+    --no-robot-model=*) BUILD_ROBOT_MODEL="OFF"; shift 1;;
+    --build-tests) BUILD_TESTING="ON"; shift 1;;
+    --help) echo "$HELP_MESSAGE"; exit 1;;
```

Comment:
I'm not very well versed in this syntax, so just remind me:
why do you have `=*)` on the first three fields but not on the last two, such as `--build-tests)`? It would seem to me all arguments have the same type now (optional flag) so should have the same syntax

Reply:
Ah shit, I forgot to remove this, it's from the old structure. Thanks